### ISSUE TITLE
PyFlyff sometimes crashes when opening alt client

### ISSUE THREAD

**Issue by Roukira** [ils94/PyFlyff#13]
As title says, sometimes the client crashes completely as soon as I open the alt client using the menu button.

**Comment by ils94** [comment]
was the icon folder inside the PyFlyff folde you downloaded?

**Comment by Roukira** [comment]
> was the icon folder inside the PyFlyff folde you downloaded?

Yes it is. I still have folder that doesn't work. The only way to solve it was to compile it myself.

**Comment by ils94** [comment]
> > was the icon folder inside the PyFlyff folde you downloaded?
> 
> Yes it is. I still have folder that doesn't work. The only way to solve it was to compile it myself.

thanks, i'll sure look into that!

**Comment by Roukira** [comment]
Yes I used the zip link provided in the reddit post.

**Comment by ils94** [comment]
> Yes I used the zip link provided in the reddit post.

interesting. I downloaded the folder and had no issue. Did your anti virus prompted ?

**Comment by Roukira** [comment]
Nop no antivirus, no missing file either from what I saw
